src/bus/address.rs

```diff
@@ -67,21 +70,21 @@ impl Ord for MmioAddress {
 impl Add<u64> for MmioAddress {
```

Comment:
Shouldn't this be `Add<MmioAddressValue>`?

Reply:
Logically it's "address + offset", so do we need MmioOffset?

Reply:
The same thing happens for PIO Address. We were thinking about making the interface uniform for both PIO and MMIO. If we don't need the `MmioAddressValue` than we should remove the `PioAddressValue`. It's also a bit confusing (the difference between `Address` and `AddressValue`).

Reply:
Seems like everybody is confused about the ...Value names, so I think it's better to make things more clear and use `MmioAddressOffset` and `PioAddressOffset` instead.